### PR TITLE
Add collectd part 2 because things got messed up a little (lot)

### PR DESCRIPTION
**Pull request by gruunday**

### CODE REVIEW

File: docs/monitoring/collectd.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ExtendedStatus on
+<IfModule mod_status.c>
+<Location /mod_status>
+  SetHandler server-status
+</Location>
+</IfModule>
+```
+
+* We also need to add this to the /etc/apache2/ports.conf
+    * <b> This may break things in /etc/apache2/sites-enabled/ </b>
```
Comment (butlerx) [inline docs/monitoring/collectd.md]: thats a big tab

File: docs/monitoring/collectd.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```
+$ pkg_add -r collectd
+```
+
+##### To install the port (source package):
+
+```
+$ cd /usr/ports/net-mgmt/collectd
+$ make clean install
+
```
Comment (butlerx) [inline docs/monitoring/collectd.md]: random line

File: docs/monitoring/collectd.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# Collectd
+
+* Full man page for collectd with full descriptions and configs:
+    * https://collectd.org/documentation/manpages/collectd.conf.5.shtml
```
Comment (butlerx) [inline docs/monitoring/collectd.md]: how big are your tabs?

**Review by butlerx — COMMENTED** [review]
see [codeclimate](https://codeclimate.com/github/redbrick/docs/pull/84)

File: docs/monitoring/collectd.md (hunk 3 above)
Comment (gruunday) [reply docs/monitoring/collectd.md]: This is hackmd. Is it 3 spaces?

**Comment by butlerx** [comment]
could you fix the markdown lint errors?

**Review by butlerx — APPROVED** [review]

**Pull request merged**